ci/github_release.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+GITHUB_NAMESPACE=${GITHUB_NAMESPACE:-seznam}
+GITHUB_PROJECT=${GITHUB_PROJECT:-slo-exporter}
+
+set -eo pipefail
+
+if [ -n "${SLO_EXPORTER_VERSION}" ]; then
+    echo "SLO_EXPORTER_VERSION=${SLO_EXPORTER_VERSION}"
+else
+    echo "SLO_EXPORTER_VERSION env variable has to be defined"
+    exit 1
+fi
+
+mkdir release
+
+# extract this particular release from changelog
+awk -v version=${SLO_EXPORTER_VERSION} '$0 ~ /## \[.+\]/ {release = $2} release == "["version"]" {print $0}' CHANGELOG.md > release/CHANGELOG
+
+# build tarballs with built binaries
+for i_file in `find build -name slo_exporter -type f`; do
+    tar -C `dirname $i_file` -czvf `echo $i_file | awk -F/ '{print "release/"$NF"."$(NF-1)".tgz"}'` `basename $i_file`
+done
+
+github-release release \
```

Comment:
I wanted to point out the `ghr` tool used in many tutorials but actually found the https://goreleaser.com/quick-start/
which looks pretty neat. It should manage the cross-build, archiving and uploading of the artifacts. 

But if this works for now, I'm pretty much OK with merging this and eventually migrating to the goreleaser in future.